.github/labeler.yml

```
@@ -0,0 +1,83 @@
+# Language labels based on file extensions
+php:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/*.php'
+
+python:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/*.py'
+
+# Component labels based on file paths
+database:
+  - changed-files:
+      - any-glob-to-any-file:
+          - 'frontend/database/**'
+          - '**/*.sql'
+
+backend:
+  - all:
+      - changed-files:
+          - any-glob-to-any-file:
+              - 'frontend/server/**'
+          - all-globs-to-all-files:
+              - '!frontend/server/src/Controllers/**'
+              - '!frontend/server/src/RabbitMQConnection.php'
+
+'UI Task':
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/*.js'
+          - '**/*.jsx'
+          - '**/*.ts'
+          - '**/*.tsx'
+          - '**/*.vue'
+
+'UX Task':
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/*.css'
+
+'CI/CD':
+  - changed-files:
+      - any-glob-to-any-file:
+          - '.github/workflows/**'
+          - '.github/actions/**'
+          - 'stuff/pipelines/**'
+
+'API Task':
+  - changed-files:
+      - any-glob-to-any-file:
+          - 'frontend/server/src/Controllers/**'
+          - 'frontend/www/api/**'
+          - 'frontend/server/src/ApiCaller.php'
+          - 'frontend/server/src/ApiUtils.php'
+          - 'frontend/server/src/Request.php'
+          - 'frontend/server/src/Exceptions/ApiException.php'
+          - 'frontend/server/cmd/APITool.php'
+
+'Cronjobs/RabbitMQ':
+  - changed-files:
+      - any-glob-to-any-file:
+          - 'stuff/cron/**'
+          - 'stuff/pipelines/rabbitmq*'
```

This pattern only matched files starting with `rabbitmq`, but the entire directory (pipelines) contains related async processing code that should be labeled together.
```suggestion
          - 'stuff/pipelines/**'
```